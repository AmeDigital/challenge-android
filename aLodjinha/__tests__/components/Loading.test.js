import React from 'react';

import renderer from 'react-test-renderer';

import Loading from '../../src/components/Loading';

describe('Component Loading', () => {
	test('should render without erros', () => {
		const tree = renderer.create(<Loading />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
