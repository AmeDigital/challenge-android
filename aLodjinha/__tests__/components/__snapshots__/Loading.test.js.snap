// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Loading should render without erros 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#5e2a84"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
